docker-compose.override.yml

```diff
@@ -18,8 +18,6 @@ services:
     ports:
      - "9006:5000"
   app:
-    environment:
-     - SYNCHRONIZER_HOST=http://research-app-api:5000
```

Comment:
How come this is moved out of the `override` file? Generally it's something we wouldn't want to apply "all the time", but rather just when running locally (I think). Maybe I'm missing an implication here.

Reply:
The way I saw it was that things in the `override` file would be things me as a user have actually needed or wanted to change during my time using the stack - the `SYNCHRONIZER_HOST` was not one of them. I am happy to move it back - what was the original motivation for the `override` file?

Reply:
Thanks for asking this way. Our production deployment instructions use the top level compose file + a deployment-specific file. The idea was to use the "override" file to contain any information that you'd want to set in deployment-specific way, when deploying on localhost. This pattern attempts to maintain a parallel structure between production deployment and local deployment, where only stuff that isn't expected to change goes in the top-level file. Rather than setting defaults and overriding in a special way for production. That's the idea, at any rate :-)

Reply:
Yeah, that makes a lot of sense - I didn't think about the deploy overrides. I moved it back now